### ISSUE TITLE
Use RPKI data as if it were route-objects

### ISSUE THREAD

**Issue by job** [pierky/arouteserver#19]
RPKI can be used to do Origin Validation and reject invalid announcements, but RPKI can also be used in context of provisioning & creation of whitelists.

If we take as example http://irrexplorer.nlnog.net/search/23.179.0.0/24 - for this prefix there is no IRR route object, but there is a RPKI ROA which states what the authorised origin AS is and this matches what is observed in the DFZ. I'd prefer to accept such a prefix from AS neighbor 6939.

In other words, we should treat RPKI ROAs as if they are IRR route objects.

Implementation suggestion: we can use `rtrsub` to generate a yml file of all such RPKI ROAs, and if `arouteserver` is expanding ASNs into route objects (after having expanded AS-SETS into ASNs), it also does a lookup in that yml file.

**Comment by pierky** [comment]
Here it is...

https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/config.d/general.yml#L223-L260

--------

OpenBGPD:
https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/examples/rich/openbgpd.conf#L147-L154

And if a route has its origin ASN authorized by an AS-SET, then...

https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/examples/rich/openbgpd.conf#L254-L259

--------

BIRD:

https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/examples/rich/bird4.conf#L21-L28

https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/examples/rich/bird4.conf#L1569-L1577

https://github.com/pierky/arouteserver/blob/566f313eb9ecb8b40247dc1cccbcd952991eda9a/examples/rich/bird4.conf#L1866-L1873


Any feedback?

**Comment by job** [comment]
Guess we'll have to test this to provide you with good feedback! :)

**Comment by pierky** [comment]
If you want to test it it's already available in the [latest pre-release version on PyPi test](https://test.pypi.org/project/arouteserver/):

`pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple --pre --upgrade arouteserver`
